code/modules/hivemind/mobs.dm

```diff
@@ -326,9 +326,82 @@
 //Higher speed than normal
 //Slighly higher speaking chance
 //Appears from hive spawner and Mechiver
-//This is simply preparing room for a new hivemob
+//Appears rarely than bomber or stinger
 //////////////////////////////////////////////////////////////////////////////
 
+/mob/living/simple_animal/hostile/hivemind/lobber
+	name = "Lobber"
+	desc = "A little cleaning robot, filled with a unknown liquid. He looks happy, a bit too happy."
+	icon_state = "lobber"
+	attacktext = "spray painted" //this shouldn't appear anyways
+	density = FALSE
+	health = 75
+	maxHealth = 75
+	melee_damage_lower = 0
+	melee_damage_upper = 0
+	speak_chance = 6
+	malfunction_chance = 10
+	ranged = TRUE
+	rapid = FALSE //Visual Studio screamed at me for trying to use FALSE/TRUE in procs below  -Wouju
+	minimum_distance = 3 //having minimum_distance too high often resulted in the mob trying to melee
+	fire_verb = "lobs goo" //reminder that the attack message is "\red <b>[src]</b> [fire_verb] at [target]!"
```

Comment:
```suggestion
	fire_verb = "lobs a ball of acid" //reminder that the attack message is "\red <b>[src]</b> [fire_verb] at [target]!"
```

lets make this match up with the description

Reply:
as mentioned in my comment bellow, the projectile itself is goo rather than acid

Reply:
changed to "lobs a sphere of goo"